tests/unit/models/reddit/test_removal_reasons.py

```diff
@@ -39,50 +41,46 @@ def test__get(self):
 
     def test_hash(self):
         reason1 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "a"), reason_id="x"
+            self.reddit, subreddit=self.reddit.subreddit("a"), id="x"
         )
         reason2 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "a"), reason_id="2"
+            self.reddit, subreddit=self.reddit.subreddit("a"), id="2"
         )
         reason3 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "b"), reason_id="1"
+            self.reddit, subreddit=self.reddit.subreddit("b"), id="1"
         )
         reason4 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "A"), reason_id="x"
-        )
-        reason5 = RemovalReason(
-            self.reddit, subreddit=Subreddit(self.reddit, "a"), reason_id="X"
+            self.reddit, subreddit=self.reddit.subreddit("A"), id="x"
         )
```

Comment:
Why was `reason5` and its test removed? That test should continue to pass without modification (and if it doesn't, it indicates that a change broke something).

Reply:
Like I said above, the equal method treated lowercase and uppercase values differently, and the RedditBase variant treats them equally.